roles/basic_app_examples/cake_php_backup_restore/tasks/main.yml

```diff
@@ -0,0 +1,81 @@
+# Deploy cakephp
+- name: Create a namespace
+  shell: "oc new-project cakephp-example"
+
+- name: Deploy cakephp app
+  shell: "oc new-app -f https://raw.githubusercontent.com/sclorg/cakephp-ex/master/openshift/templates/cakephp.json"
+
+- name: Wait for cakephp deployment
+  debug:
+    var: pod
+  until: pod and pod.status.phase == "Running"
+  retries: 20
+  delay: 30
+  vars:
+    pod: '{{ lookup("k8s",
+      kind="Pod",
+      namespace="cakephp-example",
+      label_selector="deployment=cakephp-example-1"
+    )}}'
+
+# Create a backup
+
+- name: Create ark backup of cakephp
+  k8s:
+    state : present
+    definition: "{{ lookup('file', 'create-backup-cakephp.yml')}}"
+
+- name: Wait for backup to complete
+  debug:
+    var: backup
+  until: backup and backup.status.get("phase", 0) == "Completed"
+  retries: 10
+  delay: 5
+  vars:
+    backup: '{{ lookup("k8s",
+      kind="Backup",
+      api_version="ark.heptio.com/v1",
+      namespace="heptio-ark",
+      resource_name="cakephp-backup"
+    )}}'
+
+# Simulate DR and restore
+- name: Delete cake php app
```

Comment:
Just FYI you can do this in ansible without the need to use `shell`:
```
- k8s:
    state: absent
    definition:
      apiVersion: v1
      kind: Namespace
      metadata:
        name: cakephp-example
```

This is cleaner because you get the output of the response explicitly without going through the shell